src/test/java/api/items/ItemApiExamples.java

```diff
@@ -1226,6 +1230,55 @@ public void canSearchItemsByLocation() throws Exception {
       is(mainLibraryItem.getString("id")));
   }
 
+  @Test
+  public void itemHasLastCheckInPropertiesWhenTheyAreSet() throws Exception {
+    JsonObject readingRoomItem = new JsonObject()
+      .put("id", UUID.randomUUID().toString())
+      .put("holdingsRecordId", createInstanceAndHolding().toString())
+      .put("materialTypeId", getDvdMaterialType())
+      .put("permanentLoanTypeId", getCanCirculateLoanType())
+      .put("permanentLocationId", getMainLibraryLocation())
+      .put("temporaryLocationId", getReadingRoomLocation());
+
+    JsonObject lastCheckInObj = new JsonObject();
+    lastCheckInObj.put(USER_ID_FIELD, UUID.randomUUID().toString());
```

Comment:
I think it will be better to use 'write' method here. Does it make sense?

Reply:
No as there is no 'write' method in this module.